src/main/java/org/akaza/openclinica/controller/helper/TemplateHelper.java

```diff
@@ -0,0 +1,36 @@
+package org.akaza.openclinica.controller.helper;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+
+import java.util.Random;
+
+@JsonIgnoreProperties
+public class TemplateHelper {
+
+
+    public Integer random(int digit_number) {
+        switch (digit_number) {
+            case 1:
+                return new Random().nextInt(9);
+            case 2:
+                return new Random().nextInt(99);
```

Comment:
Also the value of the argument is exclusive, so this code will never generate 99 as the random number. You should set this to 100.

Reply:
I updated the code based on your feedback and it worked great and looks much shorter , cleaner and professional. Thank you for your feedback , it was very helpful.